extension/scripts/content/home/ttHome.js

```diff
@@ -8,202 +8,254 @@ let networthInterval = false;
 	await loadDatabase();
 	console.log("TT: Home - Loading script. ");
 
-	storageListeners.settings.push(loadHome);
+	let homeFeatures = new HomeFeatures();
+	homeFeatures.init();
+
+	storageListeners.settings.push(function () {
+		homeFeatures.init();
+		featureManager.display(settings.featureDisplay);
```

Comment:
Isn't this handled in the ttGlobal.entry already?

Reply:
Yep, my bad. Fixed.